Backpack/Calendar/Classes/BPKCalendarPriceLabelCell.m

```diff
@@ -0,0 +1,87 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKCalendarPriceLabelCell.h"
+
+#import <Backpack/Spacing.h>
+#import <Backpack/Label.h>
+
+#import "BPKCalendarPriceLabelCellData.h"
+#import "BPKCalendarPriceLabelStyle.h"
+
+@interface BPKCalendarPriceLabelCell ()
+
+@property(nonatomic, strong, nonnull) BPKLabel *priceLabel;
+
+@end
+
+@implementation BPKCalendarPriceLabelCell
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        [self setup];
+    }
+    return self;
+}
+
+- (instancetype)initWithCoder:(NSCoder *)coder {
+    self = [super initWithCoder:coder];
+    if (self) {
+        [self setup];
+    }
+    return self;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self setup];
+    }
+    return self;
+}
+
+- (void)setup {
+    self.priceLabel = [[BPKLabel alloc] initWithFontStyle:BPKFontStyleTextXs];
+    self.priceLabel.numberOfLines = 2;
+    self.priceLabel.lineBreakMode = NSLineBreakByTruncatingTail;
+    [self.contentView addSubview:self.priceLabel];
+    self.priceLabel.translatesAutoresizingMaskIntoConstraints = NO;
+    self.priceLabel.textAlignment = NSTextAlignmentCenter;
+    [NSLayoutConstraint activateConstraints:@[
+        [self.priceLabel.topAnchor constraintEqualToAnchor:self.titleLabel.bottomAnchor constant:BPKSpacingSm / 2.0],
+        [self.priceLabel.leadingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor constant:BPKSpacingSm],
+        [self.priceLabel.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor constant:-BPKSpacingSm],
+        [self.priceLabel.bottomAnchor constraintLessThanOrEqualToAnchor:self.contentView.bottomAnchor]
+    ]];
+}
+
+- (void)prepareForReuse {
+    [super prepareForReuse];
+
+    self.priceLabel.text = nil;
+}
+
+- (void)configureWithData:(id)data {
+    BPKCalendarPriceLabelCellData *priceLabelCellData = (BPKCalendarPriceLabelCellData *)data;
```

Comment:
Probably worth adding an assert here to make sure the data is of the expected type

Reply:
The BPKCalendar already asserts it before it calls this method. Do you want a double assert? Or just move the assert to here?

IMO it makes more sense for the assert to be here as this method could be called (in theory) from anywhere.

Reply:
We can double assert 👍🏼 It will get compiled out in production builds so the performance implications are moot anyway. We sould safeguard so we don't crash in prod here though